modules/rustbitcoin/rust_bitcoin_lib/src/lib.rs

```diff
@@ -193,6 +194,25 @@ pub unsafe extern "C" fn rust_bitcoin_addrv2(data: *const u8, len: usize) -> *mu
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn rust_bitcoin_cmpctblocks_parse(data: *const u8, len: usize) -> i32 {
+    // Safety: Ensure that the data pointer is valid for the given length
+    let data_slice = slice::from_raw_parts(data, len);
+
+    let res = deserialize_partial::<HeaderAndShortIds>(data_slice);
+
+    match res {
+        Ok(block) => return (block.0.prefilled_txs.len() + block.0.short_ids.len()).try_into().unwrap(),
```

Comment:
I think we could return more information then simply returning this count. I know this was how we did in bitcoinfuzz v1, but I think we could have a better target here.

Reply:
Cool, move to draft?